### PR TITLE
test: add frontend e2e test (smoke test for metrics page)

### PR DESCRIPTION
**Pull request by idbeta**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] E2E test
- [ ] New feature provided
- [ ] Improve performance

### CODE REVIEW

File: web/cypress/integration/metrics/metrics-smoketest.spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('ssl smoke test', () => {
```
Comment (membphis) [inline web/cypress/integration/metrics/metrics-smoketest.spec.js:19]: ?? bad title

**Review by membphis — CHANGES_REQUESTED** [review]

File: web/cypress/integration/metrics/metrics-smoketest.spec.js (hunk 1 above)
Comment (idbeta) [reply web/cypress/integration/metrics/metrics-smoketest.spec.js]: my fault, fixed.

File: web/cypress/integration/metrics/metrics-smoketest.spec.js [hunk 2]
```diff
@@ -16,7 +16,7 @@
  */
 /* eslint-disable no-undef */
 
-context('ssl smoke test', () => {
+context('metrics page smoke test', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/metrics/metrics-smoketest.spec.js:19]:
```suggestion
context('smoke test for Metrics module', () => {
```

File: web/cypress/integration/metrics/metrics-smoketest.spec.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  const domSelectors = {
+    pageContent: '.ant-pro-page-container'
+  };
+
+  beforeEach(() => {
+    // init login 
+    cy.login();
+  })
+
+  it('visit metrics page', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/metrics/metrics-smoketest.spec.js:29]:
```suggestion
  it('should visit metrics page', () => {
```

**Review by juzhiyuan — APPROVED** [review]
Because this page is simple and only contains one iframe, then this test looks good to me.

**Review by membphis — APPROVED** [review]

**Pull request merged**